addon/model/controller.ts

```diff
@@ -126,10 +139,22 @@ export class RawEditorController implements Controller {
     return this._rawEditor.rootModelNode;
   }
 
+  get marksRegistry(): MarksRegistry {
+    return this._rawEditor.model.marksRegistry;
+  }
+
+  getMutator(): ImmediateModelMutator {
```

Comment:
is this now preferred over `model.change( (mutator) => ... )` if you have a controller?

Reply:
not yet

Reply:
but I want to provide a controller instance to commands as well, and it will replace the model instance
the change method will probably move to the controller in that case, maybe renamed to something more clear

Reply:
might want to write that down somewhere ;)

Reply:
yeah im planning to do a writeup of where we stand tomorrow